### PR TITLE
Assignment 1: Assigned 3 Task

### PR DESCRIPTION
**Pull request by sumitnegikv369**
Screenshot from 2024-03-29 11-37-03

### CODE REVIEW

File: myapp/src/day1/Question1/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Greeting from './Greeting'
+import './question1.module.css'
+
+const index = () => {
```
Comment (bhavya-goel) [inline myapp/src/day1/Question1/index.js]: simply export the components from index file

File: myapp/src/day1/Question1/question1.module.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+div{
```
Comment (bhavya-goel) [inline myapp/src/day1/Question1/question1.module.css]: remove .module from the file name

**Review by bhavya-goel — CHANGES_REQUESTED** [review]

File: myapp/src/day1/Question6/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import Button from './Button'
+
+const Question6 = () => {
+  return (
+    <div
+      style={{
+        display: 'flex',
```
Comment (bhavya-goel) [inline myapp/src/day1/Question6/index.js]: move this to it's css

**Review by bhavya-goel — CHANGES_REQUESTED** [review]

**Comment by sumitnegikv369** [comment]
updated the day 1 assignment, is there anything else needed to be change?

File: myapp/src/day1/Question1/index.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import Greeting from './Greeting'
+import './styles.css'
```
Comment (bhavya-goel) [inline myapp/src/day1/Question1/index.js]: import css in the actual component itself

**Review by bhavya-goel — CHANGES_REQUESTED** [review]

**Review by bhavya-goel — APPROVED** [review]

**Pull request merged**